docs/resources/Channel.md

```diff
@@ -741,6 +742,7 @@ Embeds are deduplicated by URL.  If a message contains multiple embeds with the
 | height?       | ?integer  | height of file (if image)                                               |
 | width?        | ?integer  | width of file (if image)                                                |
 | ephemeral? \* | boolean   | whether this attachment is ephemeral                                    |
+| waveform?     | string    |                                                                         |
```

Comment:
"duration_secs": 2.9600000381469727,
      "waveform": "AAAAAA4DAQ4ZEQQDDgASWkdjNE9WWXNLaV9hORk=",

duration_secs missing

Reply:
idk what type it is tbh
and i have only seen duration_**secs** the last months.
If others exist, I don't want to try. I hate voice messages 😆